libs/media-player/src/lib/shared/as-formatted-media-timecode-string.spec.ts

```diff
@@ -0,0 +1,58 @@
+import { asFormattedMediaTimecodeString } from './as-formatted-media-timecode-string';
+
+const validCurrentLongTime = 42343.457;
+
+const validLongTimecodeString = '11:45:43';
+
+const validCurrentShortTime = 253.99473;
+
+// NOTE: we're using Math.floor() rather than Math.max() or Math.ceil()
+const validShortTimecodeString = '00:04:13';
+
+const ninetynineHoursInSecondsPlusOne = 99 * 60 * 60 + 1;
+
+const invalidNegativeNumber = -23.88;
```

Comment:
I'm assuming that since I'm using `isNullOrUndefined()` which is tested elsewhere, I don't need to do exhaustive testing on `Infinity` and other random values because that comes along for the ride with the helper?